src/components/PersonalInformation/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { useLocation } from "react-router-dom";
+import DateInput from "../DateInput/index.jsx";
+import TextInput from "../TextInput/index.jsx";
+import styles from "./styles/styles.module.scss";
+
+const PersonalInformation = () => {
+  const location = useLocation();
+  let section = <ul></ul>;
+
+  if (location.pathname === "/") {
```

Comment:
I'm not sure about the exact reason of this `if`. Basically, the component should not care about the routing in general.

Reply:
done